debug/slides_source/debug.tex

```diff
@@ -0,0 +1,382 @@
+\documentclass[utf8x,xcolor=pdftex,dvipsnames,table]{beamer}
+\usetheme{Malmoe}  % Now it's a beamer presentation with the lisa theme!
+\setbeamertemplate{footline}[page number]
+\usecolortheme{beaver}
+\usepackage[T1]{fontenc}
+\usepackage{amsmath}
+\usepackage[utf8x]{inputenc}
+\usepackage{listings}
+
+\newcommand{\superscript}[1]{\ensuremath{^{\textrm{#1}}}}
+
+\mode<presentation>
+
+\title{Debugging}
+
+\author{%
+\footnotesize
+Frédéric Bastien \newline
+\newline
+\newline
+Institut des algorithmes d'apprentissage de Montréal \newline
+Montreal Institute for Learning Algorithms \newline
+Université de Montréal
+}
+
+\date{August 10th, Deep Learning Summer School 2015, Montréal}
+
+\setbeamertemplate{navigation symbols}{}
+\begin{document}
+
+\begin{frame}[plain]
+ \titlepage
+ \vspace{-2em}
+ \includegraphics[width=.8in]{mila.png}
+ \hfill
+ \includegraphics[width=.8in]{UdeM_logo}
+\end{frame}
+
+\section{Introduction}
+\begin{frame}{Slides and Exercices}
+  \url{http://github.com/mila-udem/summerschool2015/debug}
+\end{frame}
+
+\section{Debugging}
+\begin{frame}
+  \tableofcontents[currentsection]
+\end{frame}
+
+
+\subsection{Error message}
```

Comment:
Since this example is a bit long, it might be a good thing to announce, right at the start, what the point is (debugging shape errors) so that people approach this knowning why and how it will be useful to them.